### PR TITLE
UI/create store detail activity

### PR DESCRIPTION
**Pull request by Gijuno**

### CODE REVIEW

File: app/src/main/res/drawable/ic_comment.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+    android:width="18dp"
+    android:height="19dp"
+    android:viewportWidth="18"
+    android:viewportHeight="19">
   <path
-      android:pathData="M14.66,2.6667C14.66,1.9333 14.0667,1.3333 13.3333,1.3333H2.6667C1.9333,1.3333 1.3333,1.9333 1.3333,2.6667V10.6667C1.3333,11.4 1.9333,12 2.6667,12H12L14.6667,14.6667L14.66,2.6667ZM13.3333,2.6667V11.4467L12.5533,10.6667H2.6667V2.6667H13.3333ZM4,8H12V9.3333H4V8ZM4,6H12V7.3333H4V6ZM4,4H12V5.3333H4V4Z"
-      android:fillColor="#626262"/>
+      android:pathData="M16.4925,5.4167C16.4925,4.6375 15.825,4 15,4H3C2.175,4 1.5,4.6375 1.5,5.4167V13.9167C1.5,14.6958 2.175,15.3333 3,15.3333H13.5L16.5,18.1667L16.4925,5.4167ZM12.75,10.375H9.75V13.2083H8.25V10.375H5.25V8.9583H8.25V6.125H9.75V8.9583H12.75V10.375Z"
+      android:fillColor="#858585"/>
 </vector>
```
Comment (nutyworks) [inline app/src/main/res/drawable/ic_comment.xml:9]: Original `ic_comment.xml` is replaced. Please choose another name.

File: app/src/main/res/layout/activity_store_detail.xml [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+
+            <TextView
+                android:id="@+id/store_name"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="10dp"
+                android:layout_marginTop="16dp"
+                android:fontFamily="@font/noto_sans_kr_bold"
+                android:textColor="@color/black"
+                android:textSize="18dp"
```
Comment (nutyworks) [inline app/src/main/res/layout/activity_store_detail.xml:52]: sp

File: app/src/main/res/layout/activity_store_detail.xml [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+
+            <TextView
+                android:id="@+id/introduce_store_textview"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="24dp"
+                android:fontFamily="@font/noto_sans_kr_medium"
+                android:text="가게 소개"
+                android:textColor="@color/black"
+                android:textSize="18dp"
```
Comment (nutyworks) [inline app/src/main/res/layout/activity_store_detail.xml:73]: sp

File: app/src/main/res/layout/activity_store_detail.xml [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+                app:layout_constraintTop_toTopOf="@id/introduce_store_textview" />
+
+            <TextView
+                android:id="@+id/store_description"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="10dp"
+                android:fontFamily="@font/noto_sans_kr_regular"
+                android:textColor="@color/black"
+                android:textSize="16dp"
```
Comment (nutyworks) [inline app/src/main/res/layout/activity_store_detail.xml:92]: sp

File: app/src/main/res/layout/activity_store_detail.xml [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+
+            <TextView
+                android:id="@+id/store_picture_textview"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="16dp"
+                android:fontFamily="@font/noto_sans_kr_medium"
+                android:text="사진"
+                android:textColor="@color/black"
+                android:textSize="18dp"
```
Comment (nutyworks) [inline app/src/main/res/layout/activity_store_detail.xml:136]: sp

File: app/src/main/res/layout/activity_store_detail.xml [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+                android:layout_marginLeft="8dp"
+                android:src="@drawable/dummy_picture2"
+                app:layout_constraintBottom_toBottomOf="@id/store_picture_imageview"
+                app:layout_constraintStart_toEndOf="@id/store_picture_imageview"
+                app:layout_constraintTop_toTopOf="@id/store_picture_imageview" />
+
+            <TextView
+                android:id="@+id/store_menu_textview"
+                android:fontFamily="@font/noto_sans_kr_medium"
+                android:textSize="18dp"
```
Comment (nutyworks) [inline app/src/main/res/layout/activity_store_detail.xml:161]: sp

File: app/src/main/res/layout/activity_store_detail.xml [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+                android:id="@+id/store_menu_recyclerview"
+                android:layout_marginTop="16dp"
+                app:layout_constraintTop_toBottomOf="@id/store_menu_textview"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"/>
+
+            <TextView
+                android:id="@+id/store_comments_textview"
+                android:fontFamily="@font/noto_sans_kr_medium"
+                android:textSize="18dp"
```
Comment (nutyworks) [inline app/src/main/res/layout/activity_store_detail.xml:180]: sp

File: app/src/main/res/layout/activity_store_detail.xml [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+                app:layout_constraintTop_toBottomOf="@id/store_comments_textview"
+                android:layout_height="wrap_content">
+
+                <EditText
+                    android:background="@null"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    android:fontFamily="@font/noto_sans_kr_medium"
+                    android:textSize="12dp"
```
Comment (nutyworks) [inline app/src/main/res/layout/activity_store_detail.xml:207]: sp

**Review by nutyworks — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_store_detail.xml (hunk 2 above)
Comment (Gijuno) [reply app/src/main/res/layout/activity_store_detail.xml:52]: wow

**Review by nutyworks — APPROVED** [review]

File: app/src/main/res/layout/activity_store_detail.xml [hunk 9]
```diff
@@ -0,0 +1,242 @@
...
+                app:layout_constraintTop_toTopOf="parent"
+                tools:text="봉구스 밥버거" />
+
+            <ImageView
+                android:id="@+id/store_image_preview"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="18dp"
+                android:src="@drawable/dummy_store"
+                app:layout_constraintTop_toBottomOf="@id/store_name" />
```
Comment (nutyworks) [inline app/src/main/res/layout/activity_store_detail.xml:61]: The image should fill the entire width.

**Review by nutyworks — CHANGES_REQUESTED** [review]

**Review by nutyworks — APPROVED** [review]

**Review by nutyworks — APPROVED** [review]

**Pull request merged**